module-4/backend/week_one.md

```diff
@@ -5,18 +5,79 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+```
+That I actually really enjoy JS. Mostly, the intermission work was review of stuff we'd done in Mod 3.
+```
+
 2. What are some tools to help debug JavaScript code?
+```
+console.log();
+debugger;
+using the different tabs in the console
+```
+
 3. What are some tools you need in order to unit test your JavaScript?
+```
+Mocha will help you run your tests
+Chai will help you build your tests
+```
+
 4. What is the syntax for invoking a function?
+```
+functionName();
+```
+
 5. What is CORS?
+```
+CORS is a method to share resources between domains, e.g. a call to an API in one domain to get information for another domain.
+```
+
 6. How do we enable CORS?
+```
+... Mostly, I just added the methods I wanted access to in config/application.rb... Umm... Is there a gem? Do you just ask the internet and blindly follow their rules? Surly there's docs somewhere... ;)
```

Comment:
yup, we're using a gem in our Rails backend. we'll need to handle it in a different way when we create our node backend 👍

Reply:
Hey @case-eee, I don't know that we ever explicitly covered this... In fact, I was thinking about this over the weekend... I understand the principle of using tokens to limit access to certain requests (e.g. delete table users), but I don't understand the practical side of this. I don't see a lesson on this coming up. Will our class be able to look at this before graduation? If not, would you be willing to help me figure it out one on one?

Reply:
@squeemishly we can definitely talk about it! we don't have a class on it specifically. we briefly touch on it when you created your node backend and handled requests coming from your separate front end application. let me know when you want to chat more about it!